conanfile.py

```diff
@@ -114,142 +119,239 @@ def requirements(self):
-        if not self.options.with_postgresql_extra:
-            cmake.definitions['USERVER_FEATURE_PATCH_LIBPQ'] = 'OFF'
-        if not self.options.with_redis:
-            cmake.definitions['USERVER_FEATURE_REDIS'] = 'OFF'
-        if not self.options.with_grpc:
-            cmake.definitions['USERVER_FEATURE_GRPC'] = 'OFF'
-        if not self.options.with_clickhouse:
-            cmake.definitions['USERVER_FEATURE_CLICKHOUSE'] = 'OFF'
-        if not self.options.with_universal:
-            cmake.definitions['USERVER_FEATURE_UNIVERSAL'] = 'OFF'
-        if not self.options.with_rabbitmq:
-            cmake.definitions['USERVER_FEATURE_RABBITMQ'] = 'OFF'
-        if not self.options.with_utest:
-            cmake.definitions['USERVER_FEATURE_UTEST'] = 'OFF'
-
-        cmake.configure(build_folder=self._build_subfolder)
-        return cmake
+        tc.variables['USERVER_NAMESPACE_END'] = self.options.namespace_end
+
+        tc.variables['USERVER_FEATURE_JEMALLOC'] = self.options.with_jemalloc
+        tc.variables['USERVER_FEATURE_MONGODB'] = self.options.with_mongodb
+        tc.variables['USERVER_FEATURE_POSTGRESQL'] = self.options.with_postgresql
+        tc.variables['USERVER_FEATURE_PATCH_LIBPQ'] = self.options.with_postgresql_extra
+        tc.variables['USERVER_FEATURE_REDIS'] = self.options.with_redis
+        tc.variables['USERVER_FEATURE_GRPC'] = self.options.with_grpc
+        tc.variables['USERVER_FEATURE_CLICKHOUSE'] = self.options.with_clickhouse
+        tc.variables['USERVER_FEATURE_UNIVERSAL'] = self.options.with_universal
+        tc.variables['USERVER_FEATURE_RABBITMQ'] = self.options.with_rabbitmq
+        tc.variables['USERVER_FEATURE_UTEST'] = self.options.with_utest
+        tc.variables['USERVER_FEATURE_TESTSUITE'] = self.options.with_utest
+
+        tc.generate()
+
+        CMakeDeps(self).generate()
 
     def build(self):
-        cmake = self._configure_cmake()
+        cmake = CMake(self)
+        cmake.configure()
         cmake.build()
 
+    @property
+    def _cmake_subfolder(self):
+        return os.path.join(self.package_folder, 'cmake')
+
+
     def package(self):
         self.copy(pattern='LICENSE', dst='licenses')
 
-        self.copy(
-            pattern='*', dst='include', src='core/include', keep_path=True,
-        )
-        self.copy(
-            pattern='*', dst='include', src='shared/include', keep_path=True,
-        )
+        copy(self, pattern='*', dst=os.path.join(self.package_folder, "include"), src=os.path.join(self.source_folder, "core", "include"), keep_path=True)
+        copy(self, pattern='*', dst=os.path.join(self.package_folder, "include"), src=os.path.join(self.source_folder, "shared", "include"), keep_path=True)
 
+        if self.options.with_universal:
+            copy(self, pattern='*', dst=os.path.join(self.package_folder, "include"), src=os.path.join(self.source_folder, "univesal", "include"), keep_path=True)
```

Comment:
Too long lines, we use 80 characters limit. Isn't there anything in userver scripts to reformat python files? `make format` maybe?